README.md

```diff
@@ -1,28 +1,34 @@
 SLMM - The coding exercise for server side applicants
 =====================================================
 
-Please do not overengineer this exercise, but you are welcome to demonstrate knowledge in a field that can influence this project.
+Please do not overengineer this exercise, but at the same time try to demonstrate knowledge of good code design principles.
 
 ### Short description
-Assume we are interested in creating a smart lawn mowing machine called SLMM (smart lawn mowing machine). Your task is to create the software that will run in the SLMM itself and will be responsible for doing the following actions:
+Assume we are interested in creating a smart lawn mowing machine called SLMM (smart lawn mowing machine).
+The SLMM operates in a rectangular garden that is a grid with dimensions `Length x Width`; SLMM can move forward
+to the next grid cell or turn 90° left or right.
 
-1. Turn the SLMM 90o left or right -> This should take 2 seconds to do
+Your task is to create the software that will run in the SLMM itself and will be responsible for doing the following actions:
+
+1. Turn the SLMM 90° left or right -> This should take 2 seconds to do
 1. Move forward by one position -> This takes 5 seconds to do
 
-To emulate work being done, please use Sleep.
+To emulate work being done, please use `Sleep`.
+
+You are expected to create a web API that will accept the above commands, and execute them. During application startup, the SLMM is given a set of dimensions (passed in through config) that represent the width and length of the garden that the SLMM is in.
 
-You are expected to write a short web application that will accept the above commands, and execute them. During application startup, the SLMM is given a set of dimensions (passed in through config) that represent the width and length of the garden that the SLMM is in.
+UI is not required for this exercise, you can use Postman, curl, or similar client to access the API.
 
 Please try to not take more than 3-4 hours total on this exercise.
 
 ### Actions to support
-1. Turn Left
-1. Turn Right
-1. Move Forward
-1. Get current position
+1. Turn left
+1. Turn right
+1. Move one step forward
+1. Get current position (location in the garden and orientation)
```

Comment:
Maybe mention orientation too? Still a single method, but have it return full status of SLMM like (x, y, orientation) where orientation can be up, left, right, down?